torchvision/models/mnasnet.py

```diff
@@ -1,8 +1,11 @@
 import warnings
 
 import torch
+from torch import Tensor
 import torch.nn as nn
 from .utils import load_state_dict_from_url
+from torch.jit.annotations import Dict, List
```

Comment:
Given that we now use Python3, you can now import `Dict` and `List` from `typing`.

Reply:
Also, this is not very important as a change, but it is a nice cleanup that could be done

Reply:
Oh I thought it was for jittability reasons on those specific types. I'll update the other PRs too then!